book/src/controlling_system_execution/state-specific_dispatcher.md

```diff
@@ -74,7 +75,9 @@ impl<'a, 'b> SimpleState for CustomState<'a, 'b> {
         dispatcher_builder.add(MovePaddlesSystem, "move_paddles_system", &[]);
 
         // Build and setup the `Dispatcher`.
-        let mut dispatcher = dispatcher_builder.build();
+        let mut dispatcher = dispatcher_builder
+            .with_pool(world.read_resource::<ArcThreadPool>().clone())
```

Comment:
Could you add a quick comment explaining what is happening there? Maybe under the code in a quote block, as it is "advanced" stuff.

Reply:
Sure. I had a little blurb above but it was pretty light. Moved it below and added some more detail. How's that look to you?